pkg/policy/api/utils.go

```diff
@@ -146,7 +146,7 @@ const (
 	ForceNamespace Option = iota
 )
 
-func ResourceQualifiedName(namespace, cecName, resourceName string, options ...Option) string {
+func ResourceQualifiedName(namespace, cecName, resourceName string, updated *bool, options ...Option) string {
```

Comment:
Personally i'd prefer having the updated `bool` as secondary return parameter instead of passing it as pointer here too (like in the using functions). Even if it might makes the usage of it a little bit more complex (assignment & check that we don't override a updated `true` with `false in cases were we execute the function multiple times)

Reply:
Actually it's more about the "pattern" ("unexpected" side-effect on an input parameter) than the inconsistency with the other functions. IMO, it's even better to only have it this way in the inner function.